packages/web-client/app/components/card-pay/deposit-workflow/transaction-status/index.hbs

```diff
@@ -36,11 +36,18 @@
             <Boxel::Button @as="anchor" @size="extra-small" @href={{this.blockscoutUrl}} target="_blank" rel="noopener" data-test-blockscout-button>
               View on Blockscout
             </Boxel::Button>
+          {{else if this.errored}}
```

Comment:
Just for consistency's sake I'd prefer if this was `{{else if this.error}}` (even if the props of the actual error object aren't being accessed, like [here for example](https://github.com/cardstack/cardstack/blob/main/packages/web-client/app/components/card-pay/issue-prepaid-card-workflow/authentication/index.hbs#L9))

Reply:
Oops, didn't realize this is merged already 😅

Reply:
All good, can change this when I PR a similar change for the withdrawal workflow. Thanks for pointing this out.